copier/main.py

```diff
@@ -376,13 +382,21 @@ def jinja_env(self) -> SandboxedEnvironment:
         """
         paths = [str(self.template.local_abspath)]
         loader = FileSystemLoader(paths)
+        default_extensions = [
+            "cookiecutter.extensions.JsonifyExtension",
+            "cookiecutter.extensions.RandomStringExtension",
+            "cookiecutter.extensions.SlugifyExtension",
+            "jinja2_time.TimeExtension",
+            "jinja2_ansible_filters.AnsibleCoreFiltersExtension",
+        ]
+        extensions = default_extensions + list(self.extensions)
```

Comment:
I chose to set default extensions here, so they are *always* loaded, whatever the template writer puts in `_extensions`. But maybe you think we should let writers override these?